src/lib/modules/Document/DocumentPublicationInfo.js

```diff
@@ -0,0 +1,71 @@
+import { InfoMessage } from '@components/InfoMessage';
+import React, { Component } from 'react';
+import { Divider, Table } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import _isEmpty from 'lodash/isEmpty';
+
+export class DocumentPublicationInfo extends Component {
+  render() {
+    const { publications, documentType } = this.props;
+    if (_isEmpty(publications)) {
+      return (
+        <InfoMessage
+          title="No publication information available."
+          message="This literature has not been published in any journal."
+        />
+      );
+    }
+    return (
+      <>
+        <Divider horizontal>Publication information</Divider>
+        This {documentType.toLowerCase()} was published in:
+        {publications.map((publication, idx) => (
+          <>
+            {idx > 0 && <Divider horizontal />}
+            <Table definition key={publication.journal_title}>
```

Comment:
Does it make sense to display them all, even if there is no information about it? So for example if the `pages` information is not available we should entirely omit pages table row.

Reply:
fixed